src/gui/application.cpp

```diff
@@ -508,7 +511,13 @@ void Application::showVersion()
     stream << _theme->appName().toLatin1().constData()
            << QLatin1String(" version ")
            << _theme->version().toLatin1().constData() << endl;
-    stream << "Using Qt " << qVersion() << endl;
+#ifdef GIT_SHA1
+    stream << "Git revision " << GIT_SHA1 << endl;
+#endif
+    stream << "Using Qt " << qVersion() << ", built against Qt " << QT_VERSION_STR << endl;
+#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
+    stream << "Using '" << QSslSocket::sslLibraryVersionString() << "', built against '" << OPENSSL_VERSION_TEXT << "'" << endl;
```

Comment:
the version of openssl we are build against does not mean any sense.
Maybe what makes sens is the version Qt is built against.
So i'd leave it out.